server/src/main/java/com/cloud/servlet/ConsoleProxyServlet.java

```diff
@@ -478,7 +481,12 @@ private String composeConsoleAccessUrl(String rootUrl, VirtualMachine vm, HostVO
             param.setClientTunnelSession(parsedHostInfo.third());
         }
 
-        sb.append("/ajax?token=" + encryptor.encryptObject(ConsoleProxyClientParam.class, param));
+        if (param.getHypervHost() != null || !ConsoleProxyManager.NoVncConsoleDefault.value()) {
+            sb.append("/ajax?token=" + encryptor.encryptObject(ConsoleProxyClientParam.class, param));
+        } else {
+            sb.append("/resource/noVNC/vnc_lite.html?port=" + ConsoleProxyManager.DEFAULT_NOVNC_PORT + "&token="
```

Comment:
Why not using `vnc.html` which provides more functionalities? Briefly tested invoking it and got this extra functionalities:

![image](https://user-images.githubusercontent.com/5295080/77692518-87ecd480-6f85-11ea-9495-352e3adb4a0a.png)

Reply:
Wanted a simpler, minimalist UI, can add this when we get multi lingual keyboard working